src/components/blocks/favorite-location/FavoriteLocation.tsx

```diff
@@ -0,0 +1,9 @@
+export default function FavoriteLocation(props: { title: string }): JSX.Element {
```

Comment:
```suggestion
export default function FavoriteLocation({ title }: { title: string }): JSX.Element {
```

По идее должно вот так сработать